Saule/Http/PaginationConfig.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System.Configuration;
+
+namespace Saule.Http
+{
+    /// <summary>
+    /// Pagination configuration.  Property values can be set programmatically or be read from web.config app settings.
+    /// </summary>
+    public class PaginationConfig
```

Comment:
I'm somewhat questioning the need for this configuration. Perhaps it's good enough to pick some sensible defaults instead? We can always add it again later if there's a compelling reason.